src/main/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -159,11 +163,13 @@
             "is larger than the available memory.")
     public Boolean FORCE_GC = true;
 
-    @Argument(doc = "If true, the output records are sorted by read name. Otherwise they are unsorted.")
+    @Argument(doc = "If true, the output records are sorted by read name. Otherwise they are output in the same order " +
+            "that the data was produced on the sequencer (ordered by tile and position).")
     public Boolean SORT = true;
 
+    @Deprecated
     @Argument(doc = "Configure SortingCollections to store this many records before spilling to disk. For an indexed" +
-            " run, each SortingCollection gets this value/number of indices.")
+            " run, each SortingCollection gets this value/number of indices. Deprecated: use `MAX_RECORDS_IN_RAM`")
```

Comment:
Given the deprecation should we default this to -1 or something so we can tell which one to use?